### PR TITLE
Update Magnum CI tests for OpenStack Newton release

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Comment by vmoravec** [comment]
Build is failing at

```
 magnum.tests.functional.api.v1.test_cluster.ClusterTest.test_create_list_sign_delete_clusters[positive]
 -------------------------------------------------------------------------------------------------------

 Captured traceback-1:
 ~~~~~~~~~~~~~~~~~~~~~
     Traceback (most recent call last):
       File "/usr/lib/python2.7/site-packages/magnum/tests/functional/api/v1/test_cluster.py", line 76, in tearDown
         self._delete_cluster(cluster_id)
       File "/usr/lib/python2.7/site-packages/magnum/tests/functional/api/v1/test_cluster.py", line 119, in _delete_cluster
         self.cluster_client.wait_for_cluster_to_delete(cluster_id)
       File "/usr/lib/python2.7/site-packages/magnum/tests/functional/api/v1/clients/cluster_client.py", line 119, in wait_for_cluster_to_delete
         lambda: self.does_cluster_not_exist(cluster_id), 10, 600)
      File "/usr/lib/python2.7/site-packages/magnum/tests/functional/common/utils.py", line 93, in wait_for_condition
         "on %s and ended on %s") % (timeout, start_time, end_time))
     Exception: Timed out after 600 seconds.  Started on 1480423019.31 and ended on 1480423619.31
```

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3581,7 +3581,7 @@ function oncontroller_magnum_service_setup
             --property os_distro=fedora-atomic magnum-service-image
     fi
 
-    if ! openstack image list --f value -c Name | grep -q "^SLE12SP1-JeOS-k8s-magnum$"; then
+    if ! openstack image list --f value -c Name | grep -q "^sles-openstack-magnum-kubernetes$"; then
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: maybe you want to use $service_image_name here and just append the .qcow2 in the url line above instead (this needs to have a arch suffix anyway)

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -3595,7 +3595,7 @@ function oncontroller_magnum_service_setup
 
         openstack image create --file $service_sles_image_name \
             --disk-format qcow2 --container-format bare --public \
-            --property os_distro=opensuse SLE12SP1-JeOS-k8s-magnum
+            --property os_distro=opensuse sles-openstack-magnum-kubernetes
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: please use $service_image_name here

File: scripts/qa_crowbarsetup.sh [hunk 3]
```diff
@@ -3864,31 +3864,32 @@ function oncontroller_testsetup
...
+            safely magnum cluster-template-create --name k8s_template \
+                --image-id sles-openstack-magnum-kubernetes \
                 --keypair-id default \
                 --external-network-id floating \
                 --flavor-id m1.smaller \
                 --master-flavor-id m2.smaller \
                 --docker-volume-size 5 \
                 --network-driver flannel \
                 --coe kubernetes \
+                --master-lb-enabled \
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: why do we require the lb to be enabled?

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: Thanks, I updated it

File: scripts/qa_crowbarsetup.sh (hunk 2 above)
Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: Thanks, I updated it

File: scripts/qa_crowbarsetup.sh (hunk 3 above)
Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: This is because deprecated `bay-create` command was deploying k8s cluster with LB on front

**Pull request merged**